browser/ui/webui/brave_vpn/vpn_panel_ui.cc

```diff
@@ -73,3 +92,15 @@ void VPNPanelUI::CreatePanelHandler(
     vpn_service->BindInterface(std::move(vpn_service_receiver));
   }
 }
+
+std::unique_ptr<content::WebUIController>
+UntrustedVPNPanelUIConfig::CreateWebUIController(content::WebUI* web_ui) {
+  auto* browser_context = web_ui->GetWebContents()->GetBrowserContext();
+  if (!brave_vpn::IsBraveVPNEnabled(browser_context)) {
```

Comment:
Instead of returning `nullptr` here, overriding `WebUIConfig::IsWebUIEnabled()` seems more appropriate?

Reply:
oh yes good spot, fixed via https://github.com/brave/brave-core/compare/21d945934e55fd4600ebc5703ad8854f0a785e89..94f8c2d240bb44733a1cb115f3bc330d1016b0de